features/api_fix_execute.feature

```diff
@@ -0,0 +1,273 @@
+Feature: Fix execute API endpoints
+
+    @series.lts
+    @uses.config.machine_type.lxd-container
+    Scenario Outline: Fix execute command on invalid CVEs/USNs
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I run `pro api u.pro.security.fix.cve.execute.v1 --data '{"cves": ["CVE-1800-123456"]}'` as non-root
+        Then stdout is a json matching the `api_response` schema
+        And the json API response data matches the `cve_fix_execute` schema
+        And stdout matches regexp:
+        """
+        {"_schema_version": "v1", "data": {"attributes": {"cves_data": {"cves": \[{"description": null, "errors": \[{"error_type": "security-fix-not-found-issue", "failed_upgrades": null, "reason": "Error: CVE-1800-123456 not found."}\], "status": "error", "title": "CVE-1800-123456", "upgraded_packages": \[\]}\], "status": "error"}}, "meta": {"environment_vars": \[\]}, "type": "CVEFixExecute"}, "errors": \[\], "result": "success", "version": ".*", "warnings": \[\]}
```

Comment:
Not for this PR, but I think we need to come up with a better way of testing what we care about in json responses. These tests are hard to read and when something goes wrong, it's hard to tell which part went wrong.

Reply:
Totally agree